### ISSUE TITLE
[Bug]: Windows Linking Doesnt work

### ISSUE THREAD

**Issue by soixt** [beyondcode/herd-community#540]
### Platform

Windows

### Operating system version

Windows 11 Pro (23H2 22631.3296)

### System architecture

Windows

### Herd Version

1.0.0

### Bug description

<img width="660" alt="image" src="https://github.com/beyondcode/herd-community/assets/100382372/578cd423-4c1f-4e8c-a8d4-70a1fda227b5">

As you see on image I tried to add link and list all links and message for adding was successful but list is empty is is valet/Sites folder.

**Comment by mechelon** [comment]
Do you run this terminal with admin privileges? These are necessary to create a symlink on Windows (which `herd link` is doing).

**Comment by soixt** [comment]
> Do you run this terminal with admin privileges? These are necessary to create a symlink on Windows (which `herd link` is doing).

Yes, I run herd as admin, cmd as admin, vscode as admin for terminal inside of it, powershell as admin, powershell ise as admin.

**Comment by simonl65** [comment]
Same here.

I thought it might be something to do with creating my site from the command line, so I tried creating one from within Herd but that has the same problem.

I also can't browse either site!

**Comment by danjdewhurst** [comment]
Same issue here. Windows 11, running everything as admin etc.

**Comment by soixt** [comment]
I even tried creating a scheduled task in Windows Task Scheduler with the highest privileges, which allows you to run programs as an administrator, typically without a User Account Control (UAC) prompt, but no luck there.

**Comment by soixt** [comment]
Not sure if this would help creators, since I am unsure how is herd built but if you are using batch scripts, you can maybe try  something like this:

```bat
@echo off
REM Create symbolic link with administrative privileges if necessary

:: Check for administrative privileges
net session >nul 2>&1
if %errorLevel% == 0 (
    goto :gotAdmin
) else (
    echo Requesting administrative privileges...
    goto :UACPrompt
)

:UACPrompt
    echo Set UAC = CreateObject("Shell.Application") > "%temp%\getadmin.vbs"
    echo UAC.ShellExecute "cmd.exe", "/c %~s0", "", "runas", 1 >> "%temp%\getadmin.vbs"
    "%temp%\getadmin.vbs"
    del "%temp%\getadmin.vbs"
    exit /b

:gotAdmin
    REM Your commands here
    mklink /D "C:\Users\user\.config\herd\config\valet\Sites\symlink" "C:\Users\user\Herd\site"
    pause
    exit /b
```

**Comment by simonl65** [comment]
WOW, that's a quick turn-around! I confirm that this is fixed - thank you 🥇

**Comment by anuzpandey** [comment]
Still facing the same problem.
Herd Version: 1.0.1

**Comment by munindra-ai** [comment]
Issue is with . file name structure 
Still facing issue v-1.0.1

**Comment by anuzpandey** [comment]
@munindra-ai Yes. I checked myself with the application with and without . (dot). And hence the application folder with dot has the issue with herd link and other doesn't.

**Comment by mechelon** [comment]
Just to make sure I get this right: "application folder with dot" = "/Code/my.app" or something like that?
And folders without a dot in them work?

**Comment by anuzpandey** [comment]
@mechelon Yes.
I have a laravel application called `inventory.myproject`. -> This has the issue.
And I cloned the same repo with a different folder name `inventory`. This has no issue in it.

**Comment by munindra-ai** [comment]
@mechelon yes 
My project is something like this 
application folder with dot  :- webhook.order

In different Folder :- "WebhookOrder" is worked

**Comment by andiyudi** [comment]
![image](https://github.com/beyondcode/herd-community/assets/49513145/acccdaf9-16d5-4c19-a237-811c129cad32)
helo, i have many try to make new site with herd, but it always show like the screenshot.
how to solve it?
Thanks

**Comment by mechelon** [comment]
@andiyudi This is not related to our problem here (folder names with dot) - still, can you check if your project is inside of one of Herds paths (you can check them in the General settings tab) or linked?

@anuzpandey @munindra-ai 
I just tried to reproduce this and I can't, linking and unlinking a folder with a dot in its name works like it expected and it's accessible in the browser too:
![CleanShot 2024-04-05 at 10 25 20@2x](https://github.com/beyondcode/herd-community/assets/26432041/e0acd597-91b4-4a5b-bd64-8313cb62f1f6)

Do you get any error messages, any output that is different from mine? Does the sites work if they are inside a parked directory (Herd Paths in General settings tab)? Did you restart Herd and your Terminal since the update?

**Comment by anuzpandey** [comment]
@mechelon I noticed the issue lies with the Folder Name with spaces 😁 . I had a folder with Spaces `Web Projects` and the application within it was not registered while the folder with no spaces and the project with and without (dot) inside that directory is working fine.

`Do you get any error messages, any output that is different from mine? `

No error messages were found.
`herd link` gives the same output as yours. (Success)
But when I check in `herd links` site is not registered there. Even inside Herd App I could not see those links.
`herd unlink` works fine too. i.e the output is as yours.

`Does the sites work if they are inside a parked directory (Herd Paths in General settings tab)?`
 Yes

`Did you restart Herd and your Terminal since the update?`
Yes.

Can you please check on your machine with the project residing inside a folder with `empty spaces`?

**Comment by munindra-ai** [comment]
@mechelon 
I just tried now it works with dot but if folder contain with space it got problem 
![with space](https://github.com/beyondcode/herd-community/assets/67892209/aa3f3a65-ee85-4791-af37-1faaccdf8483)

without space
![without space](https://github.com/beyondcode/herd-community/assets/67892209/a034de55-2533-44cd-9bed-aa736c21baa1)


----
Even I had tried 
`herd park`
`herd link`

**Comment by mpociot** [comment]
This is now fixed with Herd 1.1.0
